src/wovn_interceptor.php

```diff
@@ -25,6 +25,8 @@
   list($store, $headers) = Utils::getStoreAndHeaders($_SERVER);
 
   $_ENV['WOVN_TARGET_LANG'] = $headers->lang();
+  $headers->requestOut();
```

Comment:
Looks good call in here.
But even in line 33 called `requestOut()`, is it necessary at line 33?.

Reply:
line 33 is responseOut and is different method

Reply:
line 33 is `responseOut`, not `requestOut`

Reply:
I mistook it.
Thanks.